schedview/app/scheduler_dashboard/scheduler_dashboard.py

```diff
@@ -730,9 +730,10 @@ def create_summary_widget(self):
             disabled=True,
             selectable=1,
             hidden_columns=["tier", "survey_url"],
-            pagination="remote",
-            page_size=4,
+            # pagination="remote",
```

Comment:
Given that we have the old version in git, maybe just delete these lines instead of commenting them.